src/Command/CheckAllSchemaTemplatesNamesCommand.php

```diff
@@ -94,25 +93,18 @@ private function checkSchemaTemplateNames(array $avroFiles, array &$failed = [])
     }
 
     /**
-     * @param array<int, string> $failed
      * @return array<int, string>
      */
-    private function validateNamespaceField(string $namespace, string $schemaName, array $failed): array
+    private function validateNamespaceField(string $namespace, string $schemaName): array
     {
+        $failed = [];
+
         if ('' === $namespace) {
```

Comment:
since you decided to use the extended regex, this case should already be incorporated